lib/gui/src/shapes.rs

```diff
@@ -1,27 +1,27 @@
 #![allow(missing_docs)]
 
-use crate::display::object::DisplayObject;
-use crate::display::object::DisplayObjectOps;
-use crate::display::symbol::geometry::Sprite;
-use crate::display::shape::primitive::system::ShapeSystem;
-use crate::display::world::*;
-use crate::system::web::set_stdout;
-use crate::system::web::set_stack_trace_limit;
-use crate::system::web::forward_panic_hook_to_console;
+use basegl::display::object::DisplayObject;
+use basegl::display::object::DisplayObjectOps;
+use basegl::display::symbol::geometry::Sprite;
+use basegl::display::shape::primitive::system::ShapeSystem;
+use basegl::display::world::*;
+use basegl::system::web::set_stdout;
+use basegl::system::web::set_stack_trace_limit;
+use basegl::system::web::forward_panic_hook_to_console;
 
 use nalgebra::Vector2;
 use wasm_bindgen::prelude::*;
 
-use crate::display::shape::primitive::def::*;
-use crate::display::navigation::navigator::Navigator;
+use basegl::display::shape::primitive::def::*;
+//use basegl::display::navigation::navigator::Navigator;
```

Comment:
why it is commented? Remove or leave a todo

Reply:
Adding notice to the top of the file as we agreed.